app/src/main/java/com/codesoom/assignment/util/JsonUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.codesoom.assignment.util;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.List;
+
+public class JsonUtil {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    public static String toJson(Object object) throws JsonProcessingException {
+        return objectMapper.writeValueAsString(object);
+    }
+
+    public static Task toTask(String content) throws JsonProcessingException {
+        return objectMapper.readValue(content, Task.class);
+    }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/14071105/105052543-5a908a80-5ab3-11eb-8d30-51231d7231c3.png)

```suggestion
}

// 새로운 코드
```

코드 마지막 줄에는 빈 줄이 하나가 추가되어야 합니다. 그렇지 않으면 위와 같이 GitHub에서 표시가 됩니다.

빈 줄을 추가해야하는 이유는 GitHub의 컨벤션 때문에 그렇습니다. GitHub은 변경사항을 표시할 때 변경전 부분과 변경 후 를 모두 보여줍니다. 

```
console.log('Hello');
```

만약 위와 같이 빈 줄이 없는 코드에서 아래에 코드를 추가하게 되면 Git은 다음과 같이 표시합니다.

```
- console.log('Hello');
+ console.log('Hello');
+ console.log('world!');
```
저는 단순히 아래 한 줄만 추가했는데 왜 세줄이 변경되었다고 나오는 걸까요? 그 이유는 줄바꿈 문자 `\n`가 들어갔기 때문입니다. 

```
- console.log('Hello');
+ console.log('Hello');\n
+ console.log('world');
```
만약 미리 빈 줄을 하나 추가한 상태에서 아래에 새로운 코드를 작성할 경우에는 다음과 같이 보여집니다.

```
+ console.log('world');
```
이미 \n이 추가되어 있기 때문이죠. 따라서 코드 마지막 줄에 빈 줄 하나를 항상 추가해주는게 좋습니다.

Reply:
그렇군요 ... 흥미롭네요 ! 감사합니다 :)

Reply:
기계인간님의 블로그 글을 참조하시면 좋을 것 같아요

https://johngrib.github.io/wiki/intellij/#%ED%8C%8C%EC%9D%BC-%EB%A7%88%EC%A7%80%EB%A7%89%EC%97%90-%EC%9E%90%EB%8F%99%EC%9C%BC%EB%A1%9C-1%EA%B0%9C%EC%9D%98-%EA%B0%9C%ED%96%89-%EB%AC%B8%EC%9E%90-%EC%B6%94%EA%B0%80%ED%95%98%EA%B8%B0

Reply:
감사합니다 :)